src/text_snippets.cpp

```diff
@@ -119,7 +141,7 @@ std::string snippet_library::random_from_category( const std::string &cat ) cons
     const int index = rng( 0, count - 1 );
     auto iter = iters.first;
     std::advance( iter, index );
-    return get( iter->second );
+    return expand( get( iter->second ) );
```

Comment:
It seems that the unit test wants this function to return the unexpanded snippet.

Reply:
I've updated the test to check something slightly different, and in the process I discovered:
1. it was testing two tags that were unused, `<drop_weapon>` and `<catch_up>`
2. there was a snippet using the nonexistent tag, `<catch_up>`.

I've removed the relevant snippet, and removed the old tags from the test.

As a follow-up, I'd like to write a test that iterates through every snippet and checks that any tags it references are either a special tag (like `<yrwp>`) or exist in the snippets library. That should prevent any undefined references from sneaking in.